releasenotes/notes/29894.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+issue:
+  - 29026
+
+releaseNotes:
+- |
+  **Added** `WorkloadEntry` resources will be read from all clusters in multi-cluster installations and do not need to be duplicated.
```

Comment:
What happens if they are duplicated? Since users may have this for previous versions

Reply:
Good point. 

We can either:
- Make this only process auto-registered WorkloadEntry from other clusters. 
- Add a feature flag and document this new behavior. Users will probably have to enable this, then delete their duplicated entries. I don't think there is any de-duping, I'll need to test that.